### PR TITLE
Show Chia icon in Dock

### PR DESCRIPTION
**Pull request by ChiaMineJP**
This is a working version of [Gregory's Pull Request](https://github.com/Chia-Network/chia-blockchain-gui/pull/567).
What we've learned here is that webpack only identifies files through `import` or `require`.

This PR has been submitted in a hope that reviewers can confirm it is working.

### CODE REVIEW

**Review by emlowe — APPROVED** [review]
looks fine to me - nice to get this finally resolved...

File: src/electron/main.tsx [hunk 1]
```diff
@@ -16,6 +16,9 @@ import chiaConfig from '../util/config';
 import { i18n } from '../config/locales';
 import About from '../components/about/About';
 import packageJson from '../../package.json';
+import AppIcon from '../assets/img/chia64x64.png'
+
+const appIcon = nativeImage.createFromPath(__dirname + '/' + AppIcon);
```
Comment (paninaro) [inline src/electron/main.tsx:21]: Do you know if the '/' path separator causes issues on Windows?

File: src/electron/main.tsx [hunk 2]
```diff
@@ -16,6 +16,9 @@ import chiaConfig from '../util/config';
 import { i18n } from '../config/locales';
 import About from '../components/about/About';
 import packageJson from '../../package.json';
+import AppIcon from '../assets/img/chia64x64.png'
```
Comment (paninaro) [inline src/electron/main.tsx:19]: Minor: semicolon at end?

```suggestion
import AppIcon from '../assets/img/chia64x64.png';
```

File: src/electron/main.tsx [hunk 3]
```diff
@@ -137,6 +140,10 @@ if (!handleSquirrelEvent()) {
         },
       });
 
+      if(process.platform === 'linux'){
```
Comment (paninaro) [inline src/electron/main.tsx:143]: Minor: formatting

```suggestion
if (process.platform === 'linux') {
```

File: src/electron/main.tsx (hunk 1 above)
Comment (ChiaMineJP) [reply src/electron/main.tsx]: Updated to `path.join(__dirname, AppIcon)` in the latest PR.

File: src/electron/main.tsx (hunk 2 above)
Comment (ChiaMineJP) [reply src/electron/main.tsx]: Wow. My mistake. Fixed.

File: src/electron/main.tsx (hunk 3 above)
Comment (ChiaMineJP) [reply src/electron/main.tsx]: This is fixed too.

**Review by paninaro — APPROVED** [review]

**Review by jack60612 — APPROVED** [review]
lgtm

**Pull request merged**